src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -175,40 +180,38 @@ class Accounts extends Component {
                                 this.goToRoute(`/accounts/${row.role}/${row.user_id}`);
                             }}>
                             <TableCell
-                                className="accountsCell"
-                                style={{
-                                    "marginLeft": "0px",
-                                    "paddingLeft": "0px",
-                                }}>
+                                style={{ width: 200 }}>
                                 <Grid
                                     alignItems="center"
                                     container
                                     layout="row">
                                     <Avatar
                                         style={styles(row.name).avatar}>{row.name.toUpperCase().match(/\b(\w)/g).join("")}
                                     </Avatar>
-                                    <Truncate lines={1} ellipsis={<span>...</span>}>
-                                        {row.name}
-                                    </Truncate>
+                                    {row.name}
+
                                 </Grid>
                             </TableCell>
+                            <TableCell style={{ width: 100 }}>
+                                {row.user_id}
+                            </TableCell>
                             <TableCell
-                                className="accountsCell">
+                                style={{ width: 200 }}>
                                 <Tooltip title={row.email}>
                                     <span>
-                                        {row.email.substr(0,20)}
+                                        {row.email.substr(0, 20)}
                                     </span>
                                 </Tooltip>
                             </TableCell>
                             <TableCell
```

Comment:
I would change the phone number width to `150` to allow number to be on one line.